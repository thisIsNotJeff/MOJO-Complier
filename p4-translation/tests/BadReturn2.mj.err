 line 1, column 5: warning: function may not return a value (foo)
