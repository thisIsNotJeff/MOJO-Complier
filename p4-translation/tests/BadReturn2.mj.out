	.text
BadReturn2.foo:
LABEL L.0


	.text
BadReturn2:

